src/js/tracks/text-track-settings.js

```diff
@@ -352,18 +358,18 @@ class TextTrackSettings extends Component {
     const values = this.getValues();
 
     try {
-      if (Object.getOwnPropertyNames(values).length > 0) {
-        window.localStorage.setItem('vjs-text-track-settings', JSON.stringify(values));
+      if (values) {
```

Comment:
The default return value for `getValues` is an object which is truthy, so, we'll never enter the `else`